docs/reference/cluster/stats.asciidoc

```diff
@@ -840,6 +840,72 @@ Occurrences of the built-in analyzer in selected nodes.
 `index_count`::
 (integer)
 Number of indices using the built-in analyzer in selected nodes.
+======
+
+`synonyms`::
```

Comment:
The top `synonyms` section contains information about all `synonyms` not only "inline". We need to remove "inline" here, like this:

Contains statistics about synonyms defined  in <<analysis-synonym-tokenfilter,synonym>> and <<analysis-synonym-graph-tokenfilter,synonym graph>> token filters configuration.

---
"synonyms defined as `synonyms`" this sound strange, let's drop the second "synonyms"

Reply:
After discussing with the team, I'm proposing some changes to the stats in https://github.com/elastic/enterprise-search-team/issues/5508. Happy to discuss there, I'll change the docs once that PR is merged.

Reply:
Other comments are 🔝 , adding them